canonical-web-and-design/vanilla-framework
# Add all missing icons as an "additional" set

## Done

Added all icons present in [the SVG icons folder](https://drive.google.com/drive/u/0/folders/1vBymmk6lqtXbySG_4yIPGRgec1-TSMY6).

Also updated the import script to output pattern mixins.

Fixes #3039

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/docs/examples/standalone/patterns/icons/icons-additional, see that you are happy with how the icons look
- Visit http://0.0.0.0:8101/docs/patterns/icons#additional, see that the icons are present, and not broken. Suggest class name changes if any don't make sense.

## Screenshots

![Screenshot from 2020-12-07 15-40-01](https://user-images.githubusercontent.com/2376968/101371793-12226200-38a3-11eb-815d-07357137defe.png)
